scripts/state.js

```diff
@@ -87,8 +87,9 @@ const state = new Vue({
         this.nowplaying.completion = 0
         clearInterval(this.nowplaying.tracker)
       })
-
-      this.nowplaying.song = this.songsMap[sources[0]]
+      if(Array.isArray(sources)){
+        this.nowplaying.song = this.songsMap[sources[0]]
+      }
```

Comment:
what is this for?

Reply:
If we don't add this then we will get error of title not defined.

Reply:
when sources is not array then we can't access sources[0], that's why we are getting error like "title of undefined".

Reply:
@sanket143 Thus I made this change to solve that error.

Reply:
Sorry @sanket143 this change is for issue #41 , I apologize for it and removed it from this PR